### PR TITLE
The Supermatter Resizing: How a crystal can fit into hand

### PR DESCRIPTION
**Pull request by MelokG000**
## About The Pull Request
adds supermatter sword craft made out of:
20 adamantium
2 vortex cores
actual sm
6 minutes of crafting
announcement on fourth minute
shrinking wand

first of all, you need to craft adamantium pedestal (2 min)
image

then, you should combine it with shrink wand and sm (2 min) (announcement once you craft it) (it has a gps signal)
image2

and the last one, small sm and 2 vortex cores (2 min) (dont forget about gps signal)
image1 image3

## Why It's Good For The Game
wiz needs more challenges and fun
craft is very hard and challenging, but the reward is powerfull as fuck
## Changelog
:cl: MelokGleb and KREKS
add: very hard wiz-only sm sword craft
sprite: sm sword resprite
/:cl:

### CODE REVIEW

**Comment by Admiral-Gigaton** [comment]
hey, wouldnt it make more sense for it to be a hyper noblium stand, considering its... well, an SM shard? I think I read somewhere that the SM starts out with a coating of hyper noblium while engies set it up, and hyper nob keeps SM mega stabilized when its used in it, though I guess balancing wise its hard for a wizard to get hyper noblium

**Comment by ZephyrTFA** [comment]
They say a good pr only comes around once in a blue moon.

**Review by ZephyrTFA — APPROVED** [review]

**Comment by Metekillot** [comment]
6 minutes seems a little long. Give your average greytider a crowbar and they can breach the armory in 3 minutes

**Comment by xKREKSx** [comment]
Could you at least write in the PR that I drew the sprites? :eyes:

**Comment by Admiral-Gigaton** [comment]
> What are the sources for adamantium btw?

mainly xenobio but I think fishing can spawn it

**Comment by Loafin34** [comment]
can it be either adamantium or metallic hydrogen?

**Comment by carlarctg** [comment]
nooo hypernoblium is used for sm pedestaaals not adamantiuuum or hyydrogennn that's logically inconsistent and breaks established conventiooooonssss!!!!

File: code/datums/components/crafting/structures.dm [hunk 1]
```diff
@@ -85,3 +85,26 @@
...
 	category = CAT_STRUCTURE
 	crafting_flags = CRAFT_CHECK_DENSITY
+
+/datum/crafting_recipe/adam_pedestal
+	name = "Adamantine Pedestal"
+	result = /obj/item/adamantine_pedestal
+	reqs = list(
+		/obj/item/stack/sheet/mineral/adamantine = 20,
+	)
+	time = 120 SECONDS
```
Comment (Ghommie) [inline code/datums/components/crafting/structures.dm:95]:
```suggestion
	time = 2 MINUTES
```

File: code/datums/components/crafting/structures.dm [hunk 2]
```diff
@@ -85,3 +85,26 @@
...
+	name = "Small Supermatter Crystal"
+	result = /obj/machinery/power/supermatter_crystal/small
+	reqs = list(
+		/obj/item/gun/magic/wand/shrink = 1,
+		/obj/item/adamantine_pedestal = 1,
+	)
+	machinery = list(
+		/obj/machinery/power/supermatter_crystal = CRAFTING_MACHINERY_CONSUME,
+	)
+	time = 120 SECONDS
```
Comment (Ghommie) [inline code/datums/components/crafting/structures.dm:109]:
```suggestion
	time = 2 MINUTES
```

File: code/datums/components/crafting/melee_weapon.dm [hunk 3]
```diff
@@ -201,3 +201,15 @@
...
+/datum/crafting_recipe/sm_sword
+	name = "Supermatter Sword"
+	result = /obj/item/melee/supermatter_sword
+	reqs = list(
+		/obj/item/assembly/signaler/anomaly/vortex = (MAX_CORES_VORTEX - 1),
+	)
+	machinery = list(
+		/obj/machinery/power/supermatter_crystal/small = CRAFTING_MACHINERY_CONSUME,
+	)
+	time = 120 SECONDS
```
Comment (Ghommie) [inline code/datums/components/crafting/melee_weapon.dm:214]:
```suggestion
	time = 2 MINUTES
```

**Review by Ghommie — APPROVED** [review]

File: code/datums/components/crafting/melee_weapon.dm (hunk 3 above)
Comment (Ghommie) [inline code/datums/components/crafting/melee_weapon.dm:214]:
```suggestion
	time = 2 MINUTES
```

File: code/datums/components/crafting/structures.dm (hunk 1 above)
Comment (Ghommie) [inline code/datums/components/crafting/structures.dm:95]:
```suggestion
	time = 2 MINUTES
```

File: code/datums/components/crafting/structures.dm (hunk 2 above)
Comment (Ghommie) [inline code/datums/components/crafting/structures.dm:109]:
```suggestion
	time = 2 MINUTES
```

**Review by Ghommie — CHANGES_REQUESTED** [review]
Oh, yeah, I can't make the edits if it isn't a requested change grrrrr...
EDIT: Oh nvm, that wasn't the issue, the "allow contributions from members" option was just disabled on this PR, well, no biggie.

**Comment by MoonDeer728** [comment]
> can it be either adamantium or metallic hydrogen?

no

**Pull request merged**